index.php

```diff
@@ -20,10 +20,46 @@
 if (isset($_GET['breed'])) {
     $cats = $catHydrator->createCatEntitiesArray((int)$_GET['breed']);
     foreach ($cats as $cat) {
-        $catsHtml .= '<div class="cat-image"><img src="' . $cat->getImage() . '" alt="' . $breeds[$cat->getBreed() - 1]['breed'] . '"></div>';
+        $catsHtml .= '<div class="cat-image">
+        <div class="favorite-icon-container">
+        <form action="index.php" method="post">
+        <input class="cat-id-input" name="breedID" value="' . $cat->getBreed() . '">
+        <input class="cat-id-input" name="newFavourite" value="' . $cat->getID() . '">
+        <img class="favorite-icon" src="';
+
+        $breedWanted = $_GET['breed'] - 1;
+
+        if($cat->getID() == $breeds[$breedWanted]['favourite_id']) {
+            $catsHtml .= 'images/fav-icon-full.svg';
+            } else { $catsHtml .= 'images/fav-icon-empty.svg';
+        }
+
+        $catsHtml .=  '" alt="">
+        </form>
+        </div>
+        <img src="' . $cat->getImage() . '" alt="A cat">
```

Comment:
This alt text was changed in the last story to include the cat breed, please fix